apps/dashboard/kinds/v2beta1/dashboard_spec.cue

```diff
@@ -109,8 +109,15 @@ DashboardLink: {
 	includeVars: bool | *false
 	// If true, includes current time range in the link as query params
 	keepTime: bool | *false
+	// Placement can be used to display the link somewhere else on the dashbaord other than abovethe visualisations.
+	placement?: DashboardLinkPlacement
 }
 
+// Dashboard Link placement. Defines where the link should be displayed. 
+// - "default" renders the link above the visualisations
+// - "inControlsMenu" renders the link in bottom part of the dashboard controls dropdown menu
+DashboardLinkPlacement: "default" | "inControlsMenu"
```

Comment:
I don't think we need a "default" option here. If you make the placement optional in the schema, then undefined is the assumed default IMO and should .

Reply:
Updated in https://github.com/grafana/grafana/pull/110666/commits/ad2f35471b324a3699a7c4f08c571bb46f4e0ac5 - please let me know what you think.